R/plots.R

```diff
@@ -57,12 +57,11 @@ plotObservationsCM <- function(...) {
 #'   regression results
 #' @param xrange, a user provded range for the xvalues (dates)
 #' @param verbose flag for to print for debugging
-#' 
-#' @importFrom ggplot2 ggplot geom_line ylab theme_bw
+#' @importFrom ggplot2 ggplot aes aes_string xlab geom_point geom_line ylab theme_bw scale_colour_manual scale_linetype_manual scale_shape_manual
 plotCM <- function(flux.or.conc, show.observations, load.model, finalloads, observations=NULL, 
                    composite=TRUE, linear.interpolation=FALSE, regression=TRUE,  xrange="none", 
                    dateField="Date", verbose=FALSE) {
-  
+  value <- yday<- variable <- '.ggplot.var'
```

Comment:
yday is a function, not a variable. i guess we'll ignore this because plots will be rewritten.